numba/tests/test_random.py

```diff
@@ -531,6 +531,13 @@ def test_numpy_uniform_kwargs(self):
             paramlist=[{'low': 1.5, 'high': 1e6},
                        {'low': -2.5, 'high': 1e3},
                        {'low': 1.5, 'high': -2.5}])
+        
+    def test_numpy_uniform_impl3(self):
+        self._check_any_distrib_kwargs(
+            jit_with_kwargs("np.random.uniform", ['low', 'high', 'size']),
+            get_np_state_ptr(),
+            'uniform',
+            paramlist=[{'low' : 0.0, 'high' : 1.0, 'size': None}])
```

Comment:
Hi @kc611! I applied the suggestion and ran the test locally. The test fails because of `dtype` mismatch (see attached screenshot). I think I need to fiddle around with [this method](https://github.com/numba/numba/blob/main/numba/tests/test_random.py#LL349-L358). But I am not sure. Could you help me with this?

![image](https://github.com/numba/numba/assets/48513587/94c3da41-30a3-4f85-bb5b-3744106aebf8)